### DISCUSSION TITLE
Internal server error

### DISCUSSION THREAD

**Discussion by rogersantosferreira** [domcloud/domcloud#39] [Q&A] [answered]
Hello!
I'm trying to create and delete server instances to test the domcloud service with my students. However, today when we try to create for the first time, the server is returning with unexpected internal errors.

Both the creation and the deletion are returning errors like this, with the message: "Woops! We seem to have hit a snag. Please try again later..."

After the creation error, the runner keeps running, and for some reason it seems to fail on some DNS server issue. So much so that in the check tab, in the DNS option, it is not possible to see any DNS entry pointed out, with the following error being displayed: "It seems the DNS server have problems"

Is there any way to check if we have any errors or problems with the server at the moment?

Can someone help us?
Thanks in advance!

**Comment by willnode** [comment, marked as answer]
The server which handles the DNS for `*.domcloud.io` is not responding at that time. I checked again now it works correctly.

Workaround has been set by increasing the timeout from 10s to 30s. Hope that helps.

**Reply by rogersantosferreira** [reply]
Hi Wildan!
I just tested it now and the error remains, see:
![image](https://user-images.githubusercontent.com/3109538/226587832-99c6a4d6-bc76-48a1-b941-2fc62a957376.png)
Trying to delete an instance!

Although the monitor is showing that everything is ok!
![image](https://user-images.githubusercontent.com/3109538/226588058-ab9886b5-04ee-42ed-b2e0-8d886fc72d8b.png)

Can I make some tweak to bypass this problem?

**Comment by rogersantosferreira** [comment]
Everything is working again. Thank you, Wildan!
